packager/packager.cc

```diff
@@ -1231,6 +1257,27 @@ struct PackageSpecBodyWalk {
 
         illegalNode(ctx, original);
     }
+
+private:
+    optional<core::packages::StrictDependenciesLevel> parseStrictDependenciesOption(ast::ExpressionPtr &arg) {
+        auto value_ = ast::cast_tree<ast::Literal>(arg);
+        if (!value_ || !value_->isString()) {
+            return nullopt;
+        }
+        auto value = value_->asString();
```

Comment:
```suggestion
        auto *lit = ast::cast_tree<ast::Literal>(arg);
        if (!lit || !value_->isString()) {
            return nullopt;
        }
        auto value = lit->asString();
```

Non-blocking: I generally prefer picking different names, rather than trying to approximate shadowing with `_`-suffixed names.

Also non-blocking: I prefer using `*` in the `auto` when it's a pointer, I forget why, it was something I picked up from trevor, he had a good explanation at the time and I just believed him.